std/node/_fs/_fs_chmod_test.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+const { test } = Deno;
+import { fail, assert } from "../../testing/asserts.ts";
+import { chmod, chmodSync } from "./_fs_chmod.ts";
+
+if (Deno.build.os !== "win") {
+  test({
+    name: "ASYNC: Permissions are changed (non-Windows)",
+    async fn() {
```

Comment:
Rather than using a top level `if (Deno.build.os !== "win") {` you can use 

`skip: Deno.build.os === "win"`

example: https://github.com/denoland/deno/blob/9648d3da1483ab77d9133ecd32fa240a597134eb/cli/js/tests/net_test.ts#L21

Reply:
@ry Pretty sure that functionality only exists in `unitTest` which is an internal concept not available to tests in `std`, but please correct me if I'm wrong, still feeling my way around the differences between `cli` and `std`. `Deno.test()` relies on `TestDefinition` ([link](https://github.com/denoland/deno/blob/master/cli/js/testing.ts#L18)) which doesn't yet have a `skip` capability, though this is available through `Deno.runTests()` as a string or regex test name pattern matcher I believe.  Sounds like a good addition though and a much better API than specifying a skip name or pattern in `runTests`.  I'd be happy to incorporate that if you wanted to expose `skip` in `Deno.test()` and match more closely the unitTest API?

Reply:
Ah right - sorry!